integration/tests/cook/test_basic.py

```diff
@@ -2468,3 +2468,10 @@ def test_pools_in_default_limit_response(self):
             else:
                 resp = util.get_limit(self.cook_url, limit, user)
                 self.assertFalse('pools' in resp.json())
+
+
+    def test_data_local_support(self):
+        uuid, resp = util.submit_job(self.cook_url, supports_data_locality=True)
+        self.assertEqual(201, resp.status_code, resp.text)
+        job = util.load_job(self.cook_url, uuid)
+        self.assertEqual(True, job['supports_data_locality'], job)
```

Comment:
Should we add an integration test that asserts that data-locality costs are being taken into account?

Reply:
I'm working on one in the next PR using a fake service. The closest thing I could think of in this PR was running through a match cycle in the scheduler unit tests.

Reply:
:+1: for next PR with fake service